File: foundation/lec001/b.cpp
#include<iostream>
using namespace std;
int main(){
    //cout<<"Hello";
    /*int a,b,res,res1,res2,res3;
    cout<<"enter 1st no.";
    cin>>a;
    cout<<"enter 2st no.";
    cin>>b;
    cout<<a+b<<endl;
    cout<<a*b<<endl;
    cout<<a/b<<endl;
    cout<<a/b<<endl;*/


////prime or not prime


    int no=0;

    cin>>no;
    bool b=true;
    
    for(int i=2;i*i<no;i++)
    {
    if(no%(i)==0){
        b=false;
        break;
    }
    }
    if(b)
    {
        cout<<"prime";
    }
    else
    {
        cout<<"not prime";
    }


    return 0;
}

File: foundation/lec001/b.java
import java.util.*;
public class b
{
    public static void main(String[] args)
    {
        //System.out.print("Hello world");
        /*int a=10,b=20,res;
        res=a+b;
        System.out.print(res);*/
        
        Scanner scn = new Scanner(System.in); 

        System.out.print("enter 1st no.");
        int a = scn.nextInt();
        System.out.print("enter 2nd no.");
        int b = scn.nextInt();
        //System.out.print("enter 3nd no.");
        //int c = scn.nextInt();

        //calculator


        /*int res=a+b;
        int res1=a*b;
        int res2=a-b;
        int res3=a/b;
        System.out.println(res);
        System.out.println(res1);
        System.out.println(res2);
        System.out.println(res3);*/

        //calculator in single line
        
        char res = scn.next().charAt(0);

        System.out.println(a + res + b);

        //find smallest no. in three no.s

       /* if(a<b && a<c)
        {
            System.out.println("a is smaller:" +a);
        }
        else if(b<a && b<c)
        {
            System.out.println("b is smaller:"+ b);
        }
        else
        {
            System.out.println("c is smaller:" +c);
        }*/

        //grading system

         /*if(90<mark)
        {
            System.out.println("a is smaller:" +a);
        }
        else if(b<a && b<c)
        {
            System.out.println("b is smaller:"+ b);
        }
        else
        {
            System.out.println("c is smaller:" +c);
        }*/





    } 
}